core/utils.class.inc.php

```diff
@@ -278,6 +294,52 @@ static public function GetConfigurationValue($sCode, $defaultValue = '')
 		return $value;
 	}
 
+	/**
+	 * @since 1.3.0 N°6012
+	 */
+	static public function GetCredentials() : array {
+		$sToken = Utils::GetConfigurationValue('itop_token', '');
+		if (! empty($sToken)){
```

Comment:
Don't use `empty()` to detect if a string is empty or not, as it returns true for `'0'`, use `strlen() > 0` instead (not in the documented coding standards I agree, but discussed internally a while ago. I just added it [here](https://www.itophub.io/wiki/page?id=latest:customization:coding_standards#test_if_a_string_is_empty_with_empty))
_Mind the other usages below_

Reply:
ok done

Reply:
It doesn't care much if a token never will be just `'0'`.